feature/experimental/policy/policy_vrf_selection/ate_tests/base_vrf_selection/base_vrf_selection_test.go

```diff
@@ -32,13 +32,21 @@ func TestMain(m *testing.M) {
 }
 
 const (
-	trafficDuration = 1 * time.Minute
-	sleepOnChange   = 10 * time.Second
-	plen4           = 30
-	plen6           = 126
-	vlan10          = 10
-	vlan20          = 20
-	ipipProtocol    = 4
+	trafficDuration   = 1 * time.Minute
+	sleepOnChange     = 10 * time.Second
+	plen4             = 30
+	plen6             = 126
+	vlan10            = 10
+	vlan20            = 20
+	ipipProtocol      = 4
+	ipv6ipProtocol    = 41
+	ipv4Address       = "222.222.222.222/32"
+	ATEDESTIPv4VLAN10 = "203.0.113.0/30"
```

Comment:
Doesn't look like these three "ATE" const need to be exported. Also, I don't know why "DEST" is all capitals, because it is not an acronym. Looks like these should be named ateDestIPv4VLAN10, ateDestIPv4VLAN20, and ateDestIPv6.

Reply:
This aligns to the below. Variables names changed.

*   IPv4: ATE-DEST-IPv4-VLAN10 in VLAN10
*   IPv4: ATE-DEST-IPv4-VLAN20 in VLAN20
*   IPv6: ATE-DEST-IPv6 in VLAN20

Reply:
Done in new commit.